### PR TITLE
Updates for v1.2 of the intelligence API

### PR DESCRIPTION
**Pull request by tjmw**
Support for v1.2 of the intelligence API.

Add `AppAnnie::MetaData.translate_ids` to support translating external IDs to App Annie IDs.

### CODE REVIEW

File: spec/api_fixtures/accounts.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      "last_sales_date": "2014-01-05",
+      "account_status": "OK",
+      "first_sales_date": "2012-06-15",
+      "publisher_name": "AppCo",
+      "account_name": "AppCo ITC"
+    }
+  ],
+  "prev_page": null,
+  "page_num": 1,
+  "next_page": null
```
Comment (croaky) [inline spec/api_fixtures/accounts.json]: Thoughts on sorting the keys alphabetically to help us avoid accidental duplicates / merges / overrides?

**Review by croaky — APPROVED** [review]
Looks great!

File: spec/api_fixtures/accounts.json (hunk 1 above)
Comment (tjmw) [reply spec/api_fixtures/accounts.json]: :+1: Great idea. I had just run the previous inline fixture through `jq` for the formatting like a lazy person. I think sorting would be useful.

**Comment by tjmw** [comment]
:+1: Thanks for the feedback @croaky!

**Pull request merged**